h2/src/main/org/h2/util/MathUtils.java

```diff
@@ -322,12 +322,12 @@ public static int secureRandomInt(int lowerThan) {
         return getSecureRandom().nextInt(lowerThan);
     }
 
-    private static Random getRandom() {
-        try {
-            Class.forName("java.util.concurrent.ThreadLocalRandom");
-            return ThreadLocalRandom.current();
-        } catch (ClassNotFoundException ignored) {
-        	return sRandom.get();
-        }
-    }
+	private static Random getRandom() {
+		try {
+			Class.forName("java.util.concurrent.ThreadLocalRandom");
+			return ThreadLocalRandom.current();
+		} catch (ClassNotFoundException ignored) {
+			return sRandom.get();
+		}
+	}
```

Comment:
Previous formatting was correct, this second commit made in incompatible with H2 code style.

Use four spaces for indentation.